python/cuml/test/dask/test_random_forest.py

```diff
@@ -436,6 +439,61 @@ def predict_with_json_rf_regressor(rf, x):
         np.testing.assert_almost_equal(pred, expected_pred, decimal=6)
 
 
+@pytest.mark.parametrize('estimator_type', ['regression', 'classification'])
+def test_rf_get_combined_model_right_aftter_fit(client, estimator_type):
+    max_depth = 3
+    n_estimators = 5
+    X, y = make_classification(
+        n_samples=350,
+        n_features=20,
+        n_clusters_per_class=1,
+        n_informative=10,
+        random_state=123,
+        n_classes=2
+    )
+    X = X.astype(np.float32)
+    if estimator_type == 'classification':
+        cu_rf_mg = cuRFC_mg(
```

Comment:
Many of the params here are either defaults or close to defaults. You could omit them to shrink the test and make it easier to maintain.

Reply:
sure, no problem. I copied this exactly from the existing  dask random forest tests: https://github.com/rapidsai/cuml/blob/816bb65069e2b8fd0e02be5d01eb62c5c153665a/python/cuml/test/dask/test_random_forest.py#L366-L378

Reply:
updated in https://github.com/rapidsai/cuml/pull/3388/commits/83f10b73ed5139053d6261cbeb729a60b1980836. I tried to keep a few that would have the most impact on the runtime of training, to keep the tests quick

Reply:
added a fix in https://github.com/rapidsai/cuml/pull/3388/commits/6a1f2d242c767241205edb16a1bc7dd295325c49, sorry. I got confused by the different levels of inheritance